homeassistant/components/switch/broadlink.py

```diff
@@ -115,23 +121,28 @@ def _learn_command(call):
     def _send_packet(call):
         """Send a packet."""
         packets = call.data.get('packet', [])
```

Comment:
Just use `call.data['packet']`, `call.data['repeat']` etc. Only use `get` for optional parameters without defaults.